### PR TITLE
Feature/grandpa home

### PR DESCRIPTION
**Pull request by andborges**

### CODE REVIEW

File: src/scenes/GrandpaScene.ts [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+      } else if (area.name == "areaDoor") {
+        return new GrandpaDoorArea(this, area.x!, area.y!, area.width!, area.height!)
+      } else {
+        return new Area(area.x!, area.y!, area.width!, area.height!)
+      }
+    })
+  }
+
+  createObjects() {
+    // Door
```
Comment (KevinKons) [inline src/scenes/GrandpaScene.ts]: No need for this comment, if you feel like separting the code create a function for this code

File: src/scenes/GrandpaScene.ts [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+    const doorSpawn = this.spawnPoints.get("Door")!
+    this.doorImage = this.add.image(doorSpawn.x, doorSpawn.y, "door")
+    this.physics.world.enableBody(this.doorImage, Phaser.Physics.Arcade.STATIC_BODY)
+
+    // Letter
```
Comment (KevinKons) [inline src/scenes/GrandpaScene.ts]: No need for this comment, if you feel like separting the code create a function for this code

File: src/scenes/GrandpaScene.ts [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+    this.letterOpenText = this.add
+      .text(this.letterOpenImage.x - 85, this.letterOpenImage.y - 120, letterText, {
+        fontSize: "8px",
+        color: "black",
+        wordWrap: { width: 180 },
+      })
+      .setVisible(false)
+      .setDepth(10)
+
+    // Collectibles
```
Comment (KevinKons) [inline src/scenes/GrandpaScene.ts]: No need for this comment, if you feel like separting the code create a function for this code. Example: createCollectibles

**Review by KevinKons — APPROVED** [review]

**Review by KevinKons — CHANGES_REQUESTED** [review]
Left some comments

File: src/scenes/GrandpaScene.ts (hunk 1 above)
Comment (andborges) [reply src/scenes/GrandpaScene.ts]: Done 👍

File: src/scenes/GrandpaScene.ts (hunk 2 above)
Comment (andborges) [reply src/scenes/GrandpaScene.ts]: Done 👍

File: src/scenes/GrandpaScene.ts (hunk 3 above)
Comment (andborges) [reply src/scenes/GrandpaScene.ts]: Done 👍

**Review by KevinKons — APPROVED** [review]

**Pull request merged**